UE/Application/Ports/BtsPort.cpp

```diff
@@ -87,6 +87,7 @@ void BtsPort::handleMessage(BinaryMessage msg)
         {
             logger.logError("Unknow recipient.");
             handler->handleFailedToSendSms();
+            handler->handlePeerUeBecomesUnknown();
```

Comment:
Tutaj trzeba dać jakiś mechanizm odróżniający czy sms, czy call, bo tak to te dwie metody będą się odpalać jak przyjdzie SMS i przy Call, co może doprowadzić do błędu. Jak to zaimplementujesz to przetestuj to, bo dzwonienie już działa, to wystarczy, że odpalisz BTS i 1 UE i zadzwonisz pod nieznany numer typu 112. I dobrze by było dopisać jakieś testy do BTSPort po tej zmianie.